admin_guide/overcommit.adoc

```diff
@@ -330,35 +330,34 @@ required.
 [[disabling-swap-memory]]
 === Disabling Swap Memory
 
-It is important to understand that oversubscribing the physical resources on a
-node affects resource guarantees the Kubernetes scheduler makes during pod
-placement. For example, suppose two guaranteed pods have reached their memory
-limit. Each container may start using swap. Eventually, if there is not enough
-swap space, processes in the pods can be terminated (due to the system being
-oversubscribed).
+You can disable swap by default on your nodes in order to preserve quality of
+service guarantees. Otherwise, physical resources on a node can oversubscribe,
+affecting the resource guarantees the Kubernetes scheduler makes during pod
+placement. 
```

Comment:
It is a bad thing, but some customers want to have it enabled because they used that feature to meet certain densities in v2.  unfortunately, swap being enabled means you can use other features.

@sdodson -- would it be bad for us to disable swap by default in our install, and instead write doc to discuss how it could be turned back on if desired?  is that something we can look to do in 3.4/3.5?